configure/accessibility.md

```diff
@@ -9,3 +9,5 @@ description: Configure the accessibility of TinyMCE.
 {% include configuration/a11y_advanced_options.md %}
 
 {% include configuration/iframe_aria_text.md %}
+
+{% include configuration/taborder.md %}
```

Comment:
This really feels like the wrong place for this? This isn't a configuration option so it's pretty confusing being here... I think https://www.tiny.cloud/docs/advanced/accessibility/ would be a better place for it?

Reply:
That page is now at "/accessibility/tinymce-and-screenreaders/"

Reply:
I talked to Tyler about this one, and there wasn't really a good place for it rn, so we put it here so the content is at least somewhere. I wouldn't be surprised if it moves.

Reply:
Can we please "rediscuss" that then, as this place seems quite bad to me.

Reply:
I should also add this is the type of thing that will likely get merged and then not fixed before release, so the "we'll deal with it later probably" type comment isn't really valid here.

Reply:
- [ ] Move to [here](https://github.com/tinymce/tinymce-docs/blob/5742ef1259f7ee8fe8eb8b8f83ed0c2c653a0408/initial-configuration/editor-important-options.md?plain=1#L19) from the feature/configuring-config-ref branch